package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@netdata/charts",
+  "version": "0.0.1",
+  "description": "Netdata frontend SDK and chart utilities",
+  "main": "index.js",
+  "module": "es6/index.js",
+  "jsnext:main": "es6/index.js",
+  "repository": "git@github.com:netdata/charts.git",
+  "author": "Netdata",
+  "license": "GPL-3.0",
+  "private": false,
+  "scripts": {
+    "build:cjs": "babel src --out-dir dist --copy-files",
+    "build:es6": "BABEL_ENV=es6 babel src --out-dir dist/es6 --copy-files",
+    "build:umd": "NODE_ENV=production webpack",
+    "build": "yarn build:umd && yarn build:cjs && yarn build:es6",
+    "test": "jest --config ./jest/config.js",
+    "lint": "eslint './src/**/*.js'",
+    "clean": "rimraf lib dist es coverage",
+    "prepublish": "yarn clean && yarn build"
+  },
+  "engines": {
+    "node": "12.10"
+  },
+  "keywords": [
+    "netdata",
+    "sdk",
+    "frontend",
+    "charts"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.13.16",
+    "@babel/core": "^7.13.16",
+    "@babel/plugin-proposal-object-rest-spread": "^7.13.8",
+    "@babel/plugin-transform-modules-umd": "^7.13.0",
+    "@babel/plugin-transform-runtime": "^7.13.15",
+    "@babel/preset-env": "^7.13.15",
+    "@babel/preset-react": "^7.13.13",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-module-resolver": "^4.1.0",
+    "clean-webpack-plugin": "^4.0.0-alpha.0",
+    "eslint": "^7.25.0",
+    "eslint-plugin-react": "^7.23.2",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "jest": "^26.6.3",
+    "prettier": "^2.2.1",
+    "rimraf": "^3.0.2",
+    "webpack": "^5.36.0",
+    "webpack-cli": "^4.6.0",
+    "webpack-plugin": "^1.0.5"
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "styled-components": "^5.2.3"
```

Comment:
will you use `react` here?

Reply:
React will be used in the next PRs in order to compose the UI around the chart.
The surrounding UI is: Tooltips, Annotations, Legend, Toolboxes